cookomatic_api/db/step.py

```diff
@@ -22,6 +22,7 @@ def save_step():
 
 class Step(ndb.Model):
     """Models a unit of instruction for cooking a Dish."""
+    number = ndb.IntegerProperty(required=True)
```

Comment:
@AninOnin if you want to get the number when going through Step entities, you can say 
```
for index, step in enumerate(dish.steps):
  print index
  print step
``` 
and that would output:
```
1
Step1

2
Step2

etc.
```